draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,11 +2089,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     contain any ICE credentials and MUST NOT gather any
                     candidates.</t>
 
-                    <t>For DTLS, all m= sections MUST use the
-                    certificate for the identity that has been specified
-                    for the PeerConnection; as a result, they MUST all
-                    have the same <xref target="RFC4572"></xref>
-                    fingerprint value, or this value MUST be a
+                    <t>For DTLS, all m= sections MUST use all the
+                    certificate(s) for the identity that has been
```

Comment:
suggest just  "use all the certificate(s) that have been specified for the PeerConnection"